### PR TITLE
bridger-377 Wrap schema for KV database to support cli command

### PR DESCRIPTION
**Pull request by furoxr**
#377

### CODE REVIEW

**Comment by HackFisher** [comment]
@furoxr There are conflicts after merge #407

**Comment by furoxr** [comment]
> @furoxr There are conflicts after merge #407

resolved~

**Comment by fewensa** [comment]
Please update the schema config file


**pangolin-ropsten**

| key                         | types |
| --------------------------- | ----- |
| scan.pangolin.planned       | u64   |
| scan.ropsten.check.planned  | u64   |
| scan.ropsten.redeem.planned | u64   |
| scan.ropsten.affirm.planned | u64   |
| scan.pangolin.running       | bool  |
| scan.ropsten.check.running  | bool  |
| scan.ropsten.redeem.running | bool  |
| scan.ropsten.affirm.running | bool  |
| affirm.target               | u64   |



**darwinia-ethereum**

| key                          | types |
| ---------------------------- | ----- |
| scan.darwinia.planned        | u64   |
| scan.ethereum.check.planned  | u64   |
| scan.ethereum.redeem.planned | u64   |
| scan.ethereum.affirm.planned | u64   |
| scan.darwinia.running        | bool  |
| scan.ethereum.check.running  | bool  |
| scan.ethereum.redeem.running | bool  |
| scan.ethereum.affirm.running | bool  |
| affirm.target                | u64   |

File: frame/supports/support-command-kv/src/schema.toml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[pangolin-ropsten]
+  "scan.pangolin.planned" =	"u64"
```
Comment (furoxr) [inline frame/supports/support-command-kv/src/schema.toml:2]: It seems crate toml doesn't support dotted key name yet([TOML spec v0.5.0 support this form)](https://github.com/toml-lang/toml/releases/tag/v0.5.0#/). As a workaround, we can quote-encapsulate the dotted key.

**Review by fewensa — APPROVED** [review]

**Pull request merged**